internal/consensus/reactor.go

```diff
@@ -768,7 +768,7 @@ OUTER_LOOP:
 
 		// special catchup logic -- if peer is lagging by height 1, send LastCommit
 		if prs.Height != 0 && rs.Height == prs.Height+1 {
-			if r.pickSendVote(ps, rs.LastCommit) {
+			if r.pickSendVote(ps, rs.LastPrecommits) {
 				logger.Debug("picked rs.LastCommit to send", "height", prs.Height)
 				continue OUTER_LOOP
 			}
```

Comment:
This logic will need to be changed, because if a node is lagging we should send the LastCommit, not the LastPrecommits.

Reply:
And the last commit is a commit, not a vote in tenderdash